### PR TITLE
notify directory about maintenance

### PR DESCRIPTION
**Pull request by zagy**
@flyingcircusio/release-managers

changelog?: Improve maintenance and emergency support integration (#24029)

### CODE REVIEW

**Comment by ckauhaus** [comment]
Need to talk about implications when directory is offline

**Comment by zagy** [comment]
Interesting. In fact I haven't thought about it deeply. I figure that KVM and Ceph servers do that and it wasn't a problem. But the behavior should at least be defined, like: Directory not there → retry later.

**Comment by ckauhaus** [comment]
PR should be reworked (see comments above)

File: nixos/modules/flyingcircus/packages/fcmaintenance/setup.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def run_tests(self):
     cmdclass={'test': PyTest},
     extras_require={
         'dev': test_req + ['pytest-cov'],
```
Comment (ckauhaus) [inline nixos/modules/flyingcircus/packages/fcmaintenance/setup.py]: I'd rather use "dev" for everything. Having two variants is a bit confusing, isn't it?

Comment (zagy) [inline nixos/modules/flyingcircus/packages/fcmaintenance/setup.py]: interesting. cannot remember. Keep it simple. Maybe something implicitly installed [test]? But really, cannot remember.

**Pull request closed without merging**